tests/test_asyncio/test_search.py

```diff
@@ -1530,6 +1530,30 @@ async def test_withsuffixtrie(decoded_r: redis.Redis):
         assert "WITHSUFFIXTRIE" in info["attributes"][0]["flags"]
 
 
+@pytest.mark.redismod
+@skip_ifmodversion_lt("2.10.05", "search")
+async def test_aggregations_add_scores(decoded_r: redis.Redis):
+    assert await decoded_r.ft().create_index(
+        (
+            TextField("name", sortable=True, weight=5.0),
+            NumericField("age", sortable=True),
+        )
+    )
+
+    assert await decoded_r.ft().client.hset(
```

Comment:
`decoded_r.ft().client` is redundant. You should just use `decoded_r`, which means "Redis Client with decode_responses=True"